Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ruby:2.6.6
```

Comment:
We are using Ruby 3.0 at main branch at this moment, so maybe we will need to change the image right?